hoodie-client/src/main/java/com/uber/hoodie/io/compact/HoodieRealtimeTableCompactor.java

```diff
@@ -98,20 +98,20 @@ public HoodieCompactionMetadata compact(JavaSparkContext jsc, HoodieWriteConfig
     }
 
     log.info("After filtering, Compacting " + operations + " files");
-    List<CompactionWriteStat> updateStatusMap =
+    List<HoodieCompactionWriteStat> updateStatusMap =
```

Comment:
Naming convention for everything under this package is Hoodie* so I wanted to rename this file to follow the same.